test/filewatch_test.erl

```diff
@@ -0,0 +1,76 @@
+-module(filewatch_test).
+-include_lib("eunit/include/eunit.hrl").
+
+-define(
+   rapid_events_test(Files, NEvents),
+
+   Dir = create_sandbox(?FUNCTION_NAME),
```

Comment:
Would be nice to use a Common Lisp-style `with-sandbox` approach here that ensures the sandbox is destroyed even when the test fails (since presumably the sandbox is not useful for debugging).